CMakeLists.txt

```diff
@@ -145,10 +150,11 @@ if(APPLE)
         set(OpenMP_libgomp_LIBRARY ${OpenMP_CXX_LIB_NAMES})
         set(OpenMP_libiomp5_LIBRARY ${OpenMP_CXX_LIB_NAMES})
     endif()
-    include_directories(/usr/local/include)
 else()
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+    # STD Filesystem
+    set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} stdc++fs)
```

Comment:
@hansely - need this for non macOS flow. Not necessary to link on ubuntu, but CentOS 7 & RHEL 8 has compiler versions that require linking.

Reply:
You can add this to your PR on MIVisionX & rocAL